GoogleForCreators/web-stories-wp
# Media Library: row-based gallery

## Summary

Uses the row-based gallery in the Uploads tab - (uses the react-photo-gallery library)

<img width="400" alt="Screen Shot 2020-07-07 at 1 38 17 PM" src="https://user-images.githubusercontent.com/15134432/86703387-2bef1d00-c057-11ea-90ad-987640272615.png">

## To-do

- Change `useSmallestImage` logic, some of the thumbnails end up too blurry when stretched out to full width.
- Fix scrollbar width again - maybe use https://gist.github.com/curran/0e30c621fe4fc612bf7feb0938a68e4d
- Flip `rowBasedGallery` feature flag on.

## User-facing changes

Photos in the user's uploads are arranged in rows instead of columns

## Testing Instructions


Turn on the `rowBasedGallery` feature flag and verify that the usual operations work in this gallery arrangement as well. (eg. pagination, search, insert, edit, delete)

---



Fixes #2820
